Failed to set up a develop environment in Ubuntu and MacOS

#### Summary
Ubuntu version: 22.04 LTS
When I try to set up a dev env on a virtual ubuntu machine, I have done all the tasks on this [page](https://developers.mattermost.com/contribute/developer-setup/), but when running `make run`, the `webapp` can't be built successfully. The `server` application is built and running.

#### Expected behavior
The environment is set up successfully.

#### Observed behavior (that appears unintentional) 
### Parts of errors
```
[webapp] ERROR in ./src/components/gif_picker/static/search_end.png

 [webapp] Module build failed (from ../node_modules/image-webpack-loader/index.js):

 [webapp] Error: Cannot find module 'imagemin-gifsicle'

 [webapp] Require stack:

 [webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/image-webpack-loader/index.js

 [webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack/lib/ProgressPlugin.js

 [webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack/lib/index.js

 [webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack-cli/lib/webpack-cli.js

 [webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack-cli/lib/bootstrap.js

 [webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack-cli/bin/cli.js

 [webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack/bin/webpack.js

 [webapp]     at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)

 [webapp]     at Module._load (node:internal/modules/cjs/loader:922:27)

 [webapp]     at Module.require (node:internal/modules/cjs/loader:1143:19)

 [webapp]     at require (node:internal/modules/cjs/helpers:110:18)

 [webapp]     at module.exports (/home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/image-webpack-loader/index.js:72:20)
```


```
[webapp] ERROR in ./src/fonts/Metropolis-Light.woff

[webapp] Module build failed (from ../node_modules/image-webpack-loader/index.js):

[webapp] Error: Cannot find module 'imagemin-gifsicle'

[webapp] Require stack:

[webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/image-webpack-loader/index.js

[webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack/lib/ProgressPlugin.js

[webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack/lib/index.js

[webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack-cli/lib/webpack-cli.js

[webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack-cli/lib/bootstrap.js

[webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack-cli/bin/cli.js

[webapp] - /home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/webpack/bin/webpack.js

[webapp]     at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)

[webapp]     at Module._load (node:internal/modules/cjs/loader:922:27)

[webapp]     at Module.require (node:internal/modules/cjs/loader:1143:19)

[webapp]     at require (node:internal/modules/cjs/helpers:110:18)

[webapp]     at module.exports (/home/ubuntu/Desktop/mattermost-8.0.0/webapp/node_modules/image-webpack-loader/index.js:72:20)
```

--- a/webapp/channels/package.json
+++ b/webapp/channels/package.json
@@ -158,6 +158,8 @@
     "html-webpack-plugin": "5.5.0",
     "identity-obj-proxy": "3.0.0",
     "image-webpack-loader": "8.1.0",
+    "imagemin-gifsicle": "7.0.0",
+    "imagemin-mozjpeg": "9.0.0",
     "isomorphic-fetch": "3.0.0",
     "jest": "29.7.0",
     "jest-canvas-mock": "2.5.0",

